views/admin/accessTokens.ejs

```diff
@@ -16,7 +43,7 @@
     <a 
       class="btn-outline-success btn createToken"
       style="margin-right: 100px;"
-      href="/api/accessTokens"
+      href="/admin/accessTokens"
```

Comment:
This should be a button. It doesn't need a `href` element since it already has logic to create the button on the server.

Reply:
yeah.... we'll take this up later.